server/rest/wms_dataset.py

```diff
@@ -34,15 +37,32 @@ def __init__(self):
     @loadmodel(map={'wmsSourceId': 'wmsSource'}, model='item',
                level=AccessType.READ)
     def createWmsDataset(self, wmsSource, params):
+        # Get layer legend (TODO// Include authentication in the future)
+        # Legend to be included in the metadata?
+        baseURL = wmsSource['meta']['minerva']['wms_params']['base_url']
+        parsedUrl = getUrlParts(baseURL)
+        hostName = parsedUrl.netloc
+        typeName = params['type_name']
```

Comment:
Can you change this param to the endpoint from `type_name` to `typeName`?  I know we lack consistency in our styles, but let's be consistent within an individual endpoint.

Also, can you add `typeName` to the endpoint Description and make it a required param?